vapi/library/library_item_updatesession_file.go

```diff
@@ -66,24 +73,31 @@ func (c *Manager) AddLibraryItemFile(ctx context.Context, sessionID string, upda
 }
 
 // AddLibraryItemFileFromURI adds a file from a remote URI.
```

Comment:
nit: adds a file from a remote URI to the update session and completes the update session.

Reply:
Added. I don't remember why we're calling `CompleteLibraryItemUpdateSession` within this method - we could remove that call. I checked GitHub (and bora), don't see any references to the `AddLibraryItemFileFromURI` method outside of govc. What do you think?

Reply:
Thanks! I guess it tries to provide a convenient wrap interface that can finish the file import within one call, though the name may not indicate that clearly, I think it's better to provide consistent interfaces, if this is not a pattern we typically expose APIs in govmomi, it makes sense to decouple completeUpdateSession.

Reply:
Agree it's better to decouple here, done.